.ruff.toml

```diff
@@ -0,0 +1,130 @@
+# Exclude a variety of commonly ignored directories.
```

Comment:
This whole config belongs in `pyproject.toml` under `[tool.ruff]` IMO.

Reply:
I don't want to clutter the pyproject.toml file. I would rather stick to a separate ruff.toml, if this is not a problem for someone

Reply:
For me, the appeal of `pyproject.toml` is that it eliminates most other config files and has everything in a central location. I don't think that it will become cluttered. We could just add it to the end of the file. As hopefully more and more linting excludes are removed it should also shrink in the long run.

Reply:
OK I've changed my mind. It might actually be better this way.